applications/RomApplication/costum_strategies/rom_builder_and_solver.h

```diff
@@ -375,9 +366,11 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
                             row(PhiElemental, i * mNodalDofs + k) = row(rom_nodal_basis, k);
                     }
                 }
+                double H_ROM_WEIGHT = it_el->GetValue(HROM_WEIGHT);

```

Comment:
we use capitals only for Kratos Variables.

the double here should be lowercase
h_rom_weight